### PR TITLE
[MUI-414], [MUI-415] Asset name table component

### PR DESCRIPTION
**Pull request by p1geon**
# Description

Please include a summary of the change in short form.

## Jira ticket(optional)

[414](https://minterestfinance.atlassian.net/browse/MUI-414)
[415](https://minterestfinance.atlassian.net/browse/MUI-415)

## Type of change

Please pick options that are relevant.

- [ ] Bug fix OR refactoring (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature or refactoring that would cause existing functionality to not work as expected)
image

### CODE REVIEW

File: src/components/Table/AssetName/AssetName.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
+import React, { FC, SVGProps } from 'react';
+
+import { Tooltip } from '@mui/material';
+import { styled } from '@mui/material/styles';
+import { AssetIconBlank, AssetNameLoading } from 'assets/svg';
```
Comment (gavrilin-remote) [inline src/components/Table/AssetName/AssetName.tsx:5]: remove unused icons, replace loader issets with react-content-loader

**Review by gavrilin-remote — CHANGES_REQUESTED** [review]

File: src/components/Table/AssetName/AssetName.tsx (hunk 1 above)
Comment (p1geon) [reply src/components/Table/AssetName/AssetName.tsx:5]: fixed

Comment (p1geon) [reply src/components/Table/AssetName/AssetName.tsx:5]: fixed

File: src/components/Table/AssetName/AssetName.tsx [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+  Trezor: TrezorSmallIcon,
+  Ledger: LedgerSmallIcon,
+  WalletConnect: WalletConnectSmallIcon,
+};
+
+const Container = styled('div')(containerStyles);
+const DataContainer = styled('div')(dataContainerStyles);
+const AmountWrapper = styled('div')(amountWrapperStyles);
+const AssetNameLoadingComponent = styled(({ ...props }) => (
+  <AssetNameLoading {...props} />
```
[the remark this answers is not in the document]
Comment (p1geon) [reply src/components/Table/AssetName/AssetName.tsx:44]: fixed

File: src/components/Table/AssetName/AssetName.tsx [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+}));
+
+const Amount = styled(Typography)({
+  marginLeft: 4,
+  color: '#6D7692',
+});
+
+const Title = styled(Typography)({
+  fontFamily: 'Open Sans',
+  fontWeight: 700,
```
Comment (gavrilin-remote) [inline src/components/Table/AssetName/AssetName.tsx:45]: why here 700??? in figma 600, use table1 typography variant

File: src/stories/Table/AssetName.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+  title: 'Table/Components',
+  component: AssetComponent,
+  args: {
+    mode: 'light',
+  },
+  argTypes: {
+    mode: { control: 'select', options: ['light', 'dark'] },
+    icon: {
+      control: 'select',
+      options: ['BTCIcon', 'DAIIcon', 'USDCIcon', 'USDTIcon', 'WETHIcon'],
```
[the remark this answers is not in the document]
Comment (p1geon) [reply src/stories/Table/AssetName.stories.tsx]: removed extra icons

File: src/components/Table/AssetName/AssetName.tsx (hunk 3 above)
Comment (p1geon) [reply src/components/Table/AssetName/AssetName.tsx:45]: they are different in different places in Figma. Checking which one should be applied

Comment (p1geon) [reply src/components/Table/AssetName/AssetName.tsx:45]: 600 is correct. Figma has been updated accordingly

**Review by mr-teddy-bear — APPROVED** [review]

**Review by marinakulsha — APPROVED** [review]

**Pull request merged**